aptos-move/framework/aptos-framework/sources/primary_fungible_store.move

```diff
@@ -203,7 +203,8 @@ module aptos_framework::primary_fungible_store {
     /// Mint to the primary store of `owner`.
     public fun mint(mint_ref: &MintRef, owner: address, amount: u64) acquires DeriveRefPod {
         let primary_store = ensure_primary_store_exists(owner, fungible_asset::mint_ref_metadata(mint_ref));
-        fungible_asset::mint_to(mint_ref, primary_store, amount);
+        let fa = fungible_asset::mint(mint_ref, amount);
+        fungible_asset::deposit_internal(primary_store, fa);
```

Comment:
why this change?

Reply:
This is to make sure mint can work for dispatchable token using PFS api. otherwise mint would fail because fa::mint_to calls fa::deposit which would be rejected for dispatchable token.

Reply:
I still don't get this... 
1. do you want to call fa::mint_to now since you changed the function.
2. why it would be rejected?

Reply:
Lemme change it to fa::mint_to. This is the same issue.